File: app/src/main/java/com/slapshotapps/dragonshockey/bindingAdapters/FadeInBindingAdapter.java
package com.slapshotapps.dragonshockey.bindingAdapters;

import androidx.databinding.BindingAdapter;
import android.view.View;

public class FadeInBindingAdapter {

    @BindingAdapter("fadeInView")
    public static void fadeInView(View view, boolean fadeInView) {
        if (fadeInView) {
            view.animate().alpha(1f).setDuration(500);
        } else {
            view.animate().alpha(0f).setDuration(500);
        }
    }
}

File: app/src/main/java/com/slapshotapps/dragonshockey/bindingAdapters/FadeInBindingAdapter.kt
package com.slapshotapps.dragonshockey.bindingAdapters

import android.view.View
import androidx.databinding.BindingAdapter


@BindingAdapter("fadeInView")
fun fadeInView(view: View, fadeInView: Boolean) {
  if (fadeInView) {
    view.animate().alpha(1f).duration = 500
  } else {
    view.animate().alpha(0f).duration = 500
  }
}
